### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by ferrine**
Hi, I found a strange code that can fail some time, just refactored it:)

### CODE REVIEW

File: pymc3/distributions/distribution.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def __new__(cls, name, *args, **kwargs):
         elif name is None:
             return object.__new__(cls)  # for pickle
         else:
-            raise TypeError("needed name or None but got: " + name)
+            raise TypeError("needed name or None but got: %s" % name)
```
Comment (twiecki) [inline pymc3/distributions/distribution.py:30]: Can we do new-style `"... but got: {}".format(name)` instead?

Comment (ferrine) [inline pymc3/distributions/distribution.py:30]: I don't see any reason and old-style looks more pretty in this situation

Comment (twiecki) [inline pymc3/distributions/distribution.py:30]: Fine.

**Comment by twiecki** [comment]
Thanks!

**Pull request merged**